### PR TITLE
Addition to sustainability

### PR DESCRIPTION
**Pull request by rhiaro**
Captures the essence of https://github.com/w3ctag/ethical-web-principles/pull/31 but in one sentence

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -231,6 +231,9 @@
 New web technologies should not make this situation worse.
 We will consider power consumption and the resulting emissions
 when we introduce new technologies to the web.
+This includes maximizing the lifespan of physical devices
+through maintaining backward compatibility,
```
Comment (torgo) [inline index.html]:
```suggestion
through maintaining compatibility,
```

File: index.html [hunk 2]
```diff
@@ -231,6 +231,9 @@
...
+as well as minimizing the data storage needs of applications.
```
Comment (torgo) [inline index.html]:
```suggestion
as well as minimizing data storage needs.
```

**Review by torgo — CHANGES_REQUESTED** [review]
made changes as discussed in our f2f breakout

**Review by torgo — APPROVED** [review]

**Comment by rhiaro** [comment]
Does this look okay to you, @ricjhill ?

**Comment by cynthia** [comment]
Personal thought - I think the a nuance regarding where the data is stored is lost in this summary. It reads is if the 'device' is a continued subject, while the original wording probably implied "anywhere". (in particular, I assume it implies a data warehouse/lake of some form)

**Review by cynthia — APPROVED** [review]

File: index.html [hunk 3]
```diff
@@ -231,6 +231,9 @@
 New web technologies should not make this situation worse.
 We will consider power consumption and the resulting emissions
 when we introduce new technologies to the web.
+This includes maximizing the lifespan of physical devices
+through maintaining compatibility,
+as well as minimizing data storage needs.
```
Comment (OR13) [inline index.html]:
```suggestion
as well as minimizing data storage and processing requirements.
```

**Review by hadleybeeman — APPROVED** [review]

**Pull request merged**